docs/en/guides/guide-zettelkasten.md

```diff
@@ -30,4 +30,9 @@ To add a new directory to Zettlr, hit `Cmd/Ctrl+O`, click the first toolbar butt
 
 ## Step 3: Write!
 
-Now you'll be set to use Zettlr as your Zettelkasten management system. If you want to dig deeper into how a Zettelkasten should work and what it should look like, consult the immense amount of resources on Zettelkästen, available on the web. A good starting point is [Zettelkasten.de](https://www.zettelkasten.de/). The authors have assembled a large list of (english) blog posts on how to supercharge your writing using a Zettelkasten.
+Now you'll be set to use Zettlr as your Zettelkasten management system. Two operations are important in a Zettelkasten:
+
+1. Creating an identifier to a note, so that you can link to it: You can do this by pressing `Ctrl + L` while editing any note, which will paste a generated ID to your note.
+2. Linking to other notes: Pressing `[[` while editing will display a list of notes you can link to (those who have an idenfitier in them).
```

Comment:
This is not entirely true, as the users have the ability to change the links (see for this the Zettelkasten preferences), which is why I would rephrase this to something like

> Linking to other notes: Insert the link opening (default: `[[`) while editing to display the auto-complete dropdown, from which you can choose the file to link to.

Furthermore the addendum "those who have an identifier in them" is wrong, because you can also link to files without any ID — only in this case via filename instead of the identifier.

Reply:
I am making the changes, one question though: Is there any difference between a link to a note with an ID and a link to a note without?

Reply:
Except from exactly that: no! IDs are just easier because this de-couples the content from any file name, so you can rename the file and the links will still work. If you don't plan on renaming your files, then you could live 100% without IDs!

Reply:
Ok thanks!

Pushed the changes you requested :)